src/index.js

```diff
@@ -1,8 +1,9 @@
 import React, { Component } from 'react';
 import csjs from 'csjs';
+import scopedName from 'csjs/lib/scoped-name';
```

Comment:
I had the same reaction. I glanced at CSJS and he doesn't try to *hide* it per se, but he doesn't go out of his way to make it more accessible either (like he does with `getCss`). Meh?

This could be moot if we use `DecoratedComponent.name` for the cache key rather than an actual CSJS-scoped name (see [my comment about line 19](https://github.com/tizmagik/react-csjs/pull/39#discussion_r89676145)), but I'm not sure if that's sufficient. We're not scoping the cache key currently (just using the function returned from `DecoratedComponent`), so I was thinking it might be okay but I also have the feeling I might be missing something.